starlite/app.py

```diff
@@ -680,3 +695,26 @@ async def wrapped_send(message: "Message") -> None:
 
             return wrapped_send
         return send
+
+    def update_openapi_schema(self) -> None:
+        """Update the openapi schema to reflect the route handlers registered on the app.
```

Comment:
```suggestion
        """Update the OpenAPI schema to reflect the route handlers registered on the app
```

Fixed spelling of OpenAPI here to match other docstrings